packages/mobile/src/utils/challenges.tsx

```diff
@@ -115,207 +45,138 @@ export type ChallengesParamList = {
-        ? challenges.sendFirstTipTitleAlt
-        : challenges.sendFirstTipTitle,
-    description:
-      Platform.OS === 'ios'
-        ? challenges.sendFirstTipDescriptionAlt
-        : challenges.sendFirstTipDescription,
-    shortDescription:
-      Platform.OS === 'ios'
-        ? challenges.sendFirstTipShortDescriptionAlt
-        : challenges.sendFirstTipShortDescription,
-    progressLabel: challenges.sendFirstTipProgressLabel,
-    buttonInfo: {
-      label:
+const mobileChallengeConfig: Record<ChallengeRewardID, MobileChallengeConfig> =
+  {
+    'connect-verified': {
+      icon: WhiteHeavyCheckMark,
+      buttonInfo: {
+        navigation: {
+          screen: 'AccountVerificationScreen'
+        },
+        renderIcon: (color) => <IconCheck fill={color} />,
+        iconPosition: 'right'
+      }
+    },
+    'listen-streak': {
+      icon: Headphone,
+      buttonInfo: {
+        navigation: {
+          screen: 'trending'
+        },
+        renderIcon: (color) => <IconArrow fill={color} />,
+        iconPosition: 'right'
+      }
+    },
+    'mobile-install': {
+      icon: MobilePhoneWithArrow
+    },
+    'profile-completion': {
+      icon: WhiteHeavyCheckMark
+    },
+    referrals: {
+      icon: IncomingEnvelope
+    },
+    'ref-v': {
+      icon: IncomingEnvelope
+    },
+    referred: {
+      icon: LoveLetter
+    },
+    'track-upload': {
+      icon: MultipleMusicalNotes,
+      buttonInfo: {
+        renderIcon: (color) => <IconUpload fill={color} />,
+        iconPosition: 'right'
+      }
+    },
+    'send-first-tip': {
+      icon: MoneyMouthFace,
+      title:
+        Platform.OS === 'ios'
+          ? messages.sendFirstTipTitleAlt
+          : messages.sendFirstTipTitle,
+      description: () =>
+        Platform.OS === 'ios'
+          ? messages.sendFirstTipDescriptionAlt
+          : messages.sendFirstTipDescription,
+      shortDescription:
         Platform.OS === 'ios'
-          ? challenges.sendFirstTipButtonAlt
-          : challenges.sendFirstTipButton,
-      navigation: {
-        screen: 'explore',
-        params: { screen: 'HeavyRotation' }
+          ? messages.sendFirstTipShortDescriptionAlt
+          : messages.sendFirstTipShortDescription,
+      panelButtonText:
+        Platform.OS === 'ios'
+          ? messages.sendFirstTipButtonAlt
+          : messages.sendFirstTipButton,
+      buttonInfo: {
+        navigation: {
+          screen: 'explore',
+          params: { screen: 'HeavyRotation' }
+        }
       }
-    }
-  },
-  'first-playlist': {
-    icon: Sparkles,
-    title: challenges.firstPlaylistTitle,
-    description: challenges.firstPlaylistDescription,
-    shortDescription: challenges.firstPlaylistShortDescription,
-    progressLabel: challenges.firstPlaylistProgressLabel,
-    buttonInfo: {
-      label: challenges.firstPlaylistButton,
-      navigation: {
-        screen: 'favorites'
+    },
+    'first-playlist': {
+      icon: Sparkles,
```

Comment:
This is probably my b but I think we wanted to change this icon to treble-clef across the board.
https://linear.app/audius/issue/PAY-609#comment-130e98bb

Reply:
Niiiiice catch, ty